src/services/OswServiceTaskWiFi.cpp

```diff
@@ -179,15 +185,36 @@ bool OswServiceTaskWiFi::isWiFiEnabled() {
 void OswServiceTaskWiFi::connectWiFi() {
     this->m_hostname = std::move(OswConfigAllKeys::hostname.get());
     this->m_autoAPTimeout = 0; //First reset to avoid racing conditions
-    this->m_clientSSID = std::move(OswConfigAllKeys::wifiSsid.get());
-    this->m_clientPass = std::move(OswConfigAllKeys::wifiPass.get());
+
```

Comment:
This is a lot of code added to the function -> extract to `loadCredentials(reconnectCount)`